tutorials/tree/dictionary/writeTree.cpp

```diff
@@ -0,0 +1,59 @@
+// Author: Alvaro Tolosa-Delgado CERN 07/2023
+// Author: Jorge Agramunt Ros IFIC(Valencia,Spain) 07/2023
+
+/*************************************************************************
+ * Copyright (C) 1995-2023, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include <iostream>
+
+#include <TFile.h>
+#include <TTree.h>
+#include <TTreeReader.h>
+#include <TTreeReaderValue.h>
+
+#include "data2Tree.hpp"
+
+void writeTree()
+{
+   std::unique_ptr<TFile> ofile { TFile::Open("testFile.root", "recreate") };
+   if ( nullptr == ofile ) {
+      std::cerr << " File not open." << std::endl;
+      return;
+   }
+
+   std::unique_ptr<TTree> myTree = std::make_unique<TTree>("myTree", "");
+   myDetectorData obj_for_branch1;
+   myTree->Branch("branch1.", &obj_for_branch1);
```

Comment:
just out of curiosity, is the dot at the end of the branch name needed?

Reply:
Thank you very much! I think that information is important enough to be part of the documentation :)

Reply:
I see trace of it in https://root.cern.ch/doc/master/classTTree.html